core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/workspace/WorkspaceResourceMonitor.scala

```diff
@@ -98,7 +98,7 @@ class WorkspaceResourceMonitor(
         })
     )
   } yield
-    if (jobResults.reduce((a, b) => a && b)) { // true iff all runners for the given job completed successfully
+    if (jobResults.forall(a => a)) { // true iff all runners for the given job completed successfully
```

Comment:
Why did you make this change? And does it mean that the comment should be updated?

Reply:
The behavior is exactly the same, except that it won't cause an exception on on empty collection